t/allergens_tags.t

```diff
@@ -7,7 +7,6 @@ use utf8;
 
 use Test::More;
 use Log::Any::Adapter 'TAP';
-#use Log::Any::Adapter 'TAP', filter => "none";
```

Comment:
I'd prefer to keep those commented lines: I keep uncommenting / commenting them when I debug the tests.

Reply:
You can just run `prove` with a different `TAP_LOG_FILTER`, though. Keeping them in the source leaves the risk of accidentally committing them in uncommented form. I actually removed few of these instances in this commit.

Reply:
"You can just run prove with a different TAP_LOG_FILTER"
--> how do I do that exactly?

Reply:
Just set that environment variable when running prove, ie, `TAP_LOG_FILTER=error prove -l -j9`.